vhive.go

```diff
@@ -265,28 +263,27 @@ func (s *fwdServer) FwdHello(ctx context.Context, in *hpb.FwdHelloReq) (*hpb.Fwd
 	return resp, err
 }
 
-func setupGVisorCRI() error {
+func setupGVisorCRI() {
 	lis, err := net.Listen("unix", *criSock)
 	if err != nil {
-		return fmt.Errorf("failed to listen: %v", err)
+		log.Fatalf("failed to listen: %v", err)
 	}
 
 	s := grpc.NewServer()
 
 	gvService, err := gvcri.NewGVisorService()
 	if err != nil {
-		return fmt.Errorf("failed to create firecracker service %v", err)
+		log.Fatalf("failed to create firecracker service %v", err)
```

Comment:
```suggestion
		log.Fatalf("failed to create gvisor service %v", err)
```

Reply:
Thanks for noticing, applied your suggestion!

Reply:
> It fixes log collection, but in this logs there is a fatal error `Failed to start containerd clientfailed to dial \"/run/firecracker-containerd/containerd.sock\": context deadline exceeded`. I think that's the issue for separate PR.

This happened, because AFAIU the orchestrator object (which starts `firecracker-containerd`) is only used by firecracker — adjusted the patch accordingly, now it works.